File: src/main/java/com/jinkyumpark/bookitout/search/response/library/AvailableLibrary.java
package com.jinkyumpark.bookitout.search.response.library;

import lombok.Builder;
import lombok.Getter;

@Builder
@Getter
public class AvailableLibrary {
    private String code;
    private String name;
    private String address;
    private String libraryLink;
    private String bookLink;
}

File: search/src/main/kotlin/com/jinkyumpark/search/library/response/AvailableLibrary.kt
package com.jinkyumpark.search.library.response

data class AvailableLibrary(
    val code: String,
    val name: String,
    val address: String,
    val libraryLink: String,
    var bookLink: String?,
) {
    constructor(
        code: String, name: String, address: String, libraryLink: String
    ) : this(code, name, address, libraryLink, null)
}